test/e2e_node/jenkins/benchmark/jenkins-benchmark.properties

```diff
@@ -0,0 +1,9 @@
+GCE_HOSTS=
+GCE_IMAGE_CONFIG_PATH=test/e2e_node/jenkins/benchmark/benchmark-config.yaml
+GCE_ZONE=us-central1-f
+GCE_PROJECT=k8s-jkns-ci-node-e2e
+CLEANUP=true
+GINKGO_FLAGS='--skip="\[Flaky\]"'
+SETUP_NODE=false
```

Comment:
This parameter is really confusion. Actually we do bring up new node for testing. Anyway, just a comment, not introduced by your pr.